app.js

```diff
@@ -1,15 +1,28 @@
 const express = require("express");
-const { getTopics, getArticles } = require('./controllers')
+const { getTopics, getArticles, getArticleById } = require("./controllers");
 
 const app = express();
+app.use(express.json());
```

Comment:
We don't need to use express.json at this point, so this can be removed for now 👍